gatsby-config.js

```diff
@@ -1,34 +1,41 @@
 module.exports = {
   siteMetadata: {
-    title: `Gatsby Default Starter`,
-    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
-    author: `@gatsbyjs`,
+    title: `UFV Computer Science Association`,
```

Comment:
Whoops; Looks like I put the wrong name in for the site
Should be `Computing Student Association` not `Computer Science Association`

Reply:
Resolved this with [this commit](https://github.com/ufvcsa/ufvcsa.github.io/pull/16/commits/d1535ce707264cece4d136ef29cce271997d0d65)